requirements/system/systemmodel/system-model.tex

```diff
@@ -13,9 +13,11 @@ \subsection{System Model}
   \centering
   \includegraphics[width=\linewidth]{./system/systemmodel/images/accessjob.png}
   \caption{This diagram depicts how a user accesses his running job.
-  After starting the job, the CS returns WC ip and container id.
-  Using this information the user can connect to the container executing his job at any time.
-  Only the user that started the job and administrators can access the job.}
+  The user does not have direct connection with the WC.
+  The user establishes a SSH connection to the server.
+  Doing that the user also authenticates himself to the server.
```

Comment:
How about:

This connection authenticates the user.
It is also used to transmit the name of the job that the user wants to connect to.
The server looks up which work machine the user's job is running on and forwards the SSH connection to that work machine.
Finally, the SSH connection to the work machine is forwarded to the container running the user's job.

Reply:
Thank you, i will only change that for the job it is not necessary  to be a running one.